Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
Thanks @mikemcquaid! That makes sense. This was my first time using `git rebase`.

**RE: referencing the Ruby 1.8 docs for `chmod` and @DomT4's comment**
At this point, does it make more sense to leave the `system` call as-is?

If it is indeed OK to leave the `system` call behind, is it more correct to update the Ruby `chmod` call to the following?
`chmod 0001, "genautomake.sh"`

Thanks!